sklearn/cluster/k_means_.py

```diff
@@ -724,7 +734,7 @@ class KMeans(BaseEstimator, ClusterMixin, TransformerMixin):
 
     def __init__(self, n_clusters=8, init='k-means++', n_init=10, max_iter=300,
                  tol=1e-4, precompute_distances='auto',
-                 verbose=0, random_state=None, copy_x=True, n_jobs=1):
+                 verbose=False, random_state=None, copy_x=True, n_jobs=1):
```

Comment:
On 01/13/2015 01:11 PM, ragv wrote:
>
>     I prefer integer verbosity
>
> Just out of curiosity... Could I know why ? :)
>
>
Because then you can have degrees.
Like 1 being slightly verbose and 10 being very verbose...